pages/docs/reference/multiplatform.md

```diff
@@ -111,7 +112,7 @@ repositories {
 
 dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    expectedBy project(":")
+    expectedBy project(":common")
```

Comment:
This sample shows a project layout where the root project is the common one, with platform modules being its subprojects. Why should we change this?